RESIST theme

zarino opened the pull request.

zarino committed:
[WIP] "RESIST" theme

zarino committed:
Add Shocal dashboard layout template

TheAdamGalloway commented on resist/shocal-dashboard.js:
```diff
@@ -0,0 +1,51 @@
+  methods: {
+    fetchLatest() {
+      fetch(
+        "https://www.shocal.org/developers/apis/peepl/v1/data?apikey=Xau08wz65Sd12BaX5lU6B1W0lHyMD52167q5i"
```
@zarino I'm not sure that this should be published in the public repo. Any ideas how we can store it more securely? And I will get Max to cycle the key.

zarino replied:
Oh no! I figured it was ok, as [you committed it here](https://github.com/itsaboutpeepl/itsaboutpeepl.com/commit/117067b2ded87187c390d81e2c87ddd00791fad2) on 2nd Feb.

I guess ideally we should store this in either a WordPress `option` (yuck) or in an environment variable on the server.

Might not be worth doing either, though, if you’d rather just get this all set up in a self-contained mini site anyway.

TheAdamGalloway committed:
Remove Shocal dashboard (moved into own site)

zarino committed:
PHP colour palette fix

I was getting errors without this, for some reason.

zarino committed:
Remove fussy padding rule from top-level block groups

This made _some_ sense in the old Peepl theme, but has just
been getting in the way in the new Resist theme so far.

zarino committed:
Improve script/deploy

Can now deploy to both staging and production (staging by default),
and supports -h / --help for showing help.

zarino committed:
Add utility classes for text alignment and overflow

Also, more weights of Work Sans, which was a change we made ages ago
but forgot to commit.
